admin-dev/themes/new-theme/js/pages/product/product-map.js

```diff
@@ -24,6 +24,7 @@
  */
 
 const combinationListId = '#combination_list';
+const attachmentsBlockId = '#product_options_attachments';
```

Comment:
Why do you put it in a constant ?

Reply:
I didn't actually, julius started this branch initially
Though it can be useful if used in the other selector, so I will change this